databases/week3/hyf_lesson3.sql

```diff
@@ -0,0 +1,196 @@
+CREATE DATABASE hyf_lesson3;
+USE hyf_lesson3;
+SET NAMES 'utf8mb4';
+CREATE TABLE `meal`(
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT NULL DEFAULT NULL,
+    `location` VARCHAR(255) NOT NULL,
+    `when` DATETIME NOT NULL,
+    `max_reservations ` INT(10) UNSIGNED NOT NULL,
+    `price` DECIMAL NOT NULL,
+    `created_date` DATE NOT NULL,
+    PRIMARY KEY(`created_date`)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `reservation`(
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    contact_name ` VARCHAR(255) NOT NULL,
+    ` contact_email ` VARCHAR(255) NOT NULL,
+    ` contact_phonenumber ` VARCHAR(255) NULL,
+    ` created_date ` DATE NOT NULL,
+     ` number_of_guests ` INT(10) UNSIGNED NOT NULL,
+    ` meal_id ` INT(10) UNSIGNED NOT NULL,
+    PRIMARY KEY(` id `),
+    CONSTRAINT ` fk_meal ` FOREIGN KEY (` meal_id `) REFERENCES ` meal `(` id `) ON DELETE CASCADE
```

Comment:
If we delete all reservations for a given meal, the meal entry will also be deleted, is that what you want? (It is ok, just checking ;) )
Also, what happens if you update the id of the meal?